src/event.ts

```diff
@@ -219,6 +223,8 @@ interface EventActorUpdate extends FieldsBase {
 	targetable?: boolean
 	/** Current attributes (stats) */
 	attributes?: AttributeValue[]
+	/** Auras on start of pull */
```

Comment:
introducing a new actor field exclusively for start of pull logic seems like a leaky abstraction.
we already have `statusApply` events, why can't we use those?